src/crepe/Component.h

```diff
@@ -45,6 +50,8 @@ class Component {
 	 * \return The maximum number of instances for this component
 	 */
 	virtual int get_instances_max() const { return -1; }
+	//! Only ComponentManager needs to know the max instance count
+	friend class ComponentManager;
```

Comment:
kan je niet bij de eerste friend class ComponentManager zetten dat de ComponentManager alleen components mag maken en alleen van de get_instances_max() mag afweten inplaats van deze twee keer er bij zetten?

Reply:
Ik zie dat `Component::get_instances_max` public is geworden dus ik heb deze twee regels compleet weg gehaald.